### ISSUE TITLE
Error: Cannot find module

### ISSUE THREAD

**Issue by willin** [restify/node-restify#777]
```
{ [Error: Cannot find module './build/Release/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/default/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
```

io.js v1.5.1
restify v2.8.5

This Error is not shown in v2.8.4

**Comment by micahr** [comment]
Dtrace is optional, so if isn't installed, you can still continue using restify. You can try to manually install it via npm: `npm install --loglevel verbose dtrace-provider` to see if there is an error installing it. 

If there is an error using dtrace, you might need to file an issue with them.

**Comment by cs-koneko** [comment]
It's fixed with dtrace-provider 0.5.0 but it looks like the message it's comming from bunyan (trentm/node-bunyan#258)...

**Comment by micahr** [comment]
Makes sense. Thanks @cs-koneko for finding the source.
